server/impl/src/main/java/com/walmartlabs/concord/server/process/queue/FilterUtils.java

```diff
@@ -76,7 +76,7 @@
 
     public static List<ProcessFilter.JsonFilter> parseJson(String paramName, UriInfo uriInfo) {
         return uriInfo.getQueryParameters().entrySet().stream()
-                .filter(e -> isParam(e.getKey(), paramName))
+                .filter(e -> (isParam(e.getKey(), paramName) && isExist(getValue(e))))
```

Comment:
may be not send `requeierments` with empty value?
looks like `JsonFilter.value is @Nullable` and value can be null.

Reply:
then, should i have check [here](https://github.com/walmartlabs/concord/blob/45593c7b4f9d8b70b805ba88b0c4d9adf45a890a/k8s/agent-operator/src/main/java/com/walmartlabs/concord/agentoperator/processqueue/ProcessQueueClient.java#L55) for null and construct url?